src/proto/net.proto

```diff
@@ -3,17 +3,31 @@ syntax = "proto2";
 package proto;
 
 message ServerMessage {
+  optional bool status = 1;
```

Comment:
I find that a lot of messages I want to send are custom data types + a boolean status of success/failure.  Rather than creating a wrapper message to encapsulate both like:

```
message ConstructMessage {
  optional Construct construct = 1;
  optional bool status = 2;
}
```
I opted to create a universal success flag that can be used by any message type.  I'll add a comment explaining this.